ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -995,6 +1020,27 @@ def test_eigvals(dtype_x, test_flags, backend_fw, fn_name):
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.general_inner_product",
+    data=_generate_general_inner_product_args(),
+    test_with_out=st.just(False),
```

Comment:
Adding the `out` argument to `ivy.sum` function in the implementation as I have suggested should fix any failing tests because of the `out` argument. Let's remove this :)